proto/meta.proto

```diff
@@ -238,6 +238,21 @@ message ListActorStatesResponse {
   repeated ActorState states = 1;
 }
 
+enum ThrottleTarget {
+  SOURCE_UNSPECIFIED = 0;
```

Comment:
Nit: should follow `<ENUM_NAME>_UNSPECIFIED` convention:

```suggestion
  THROTTLE_TARGET_UNSPECIFIED = 0;
```

Reply:
Actually why do we need `ThrottleTarget` 🤔 seems like we only have 1 kind, which is MV.
Is it for compatibility, in case we have more kinds of throttle targets in the future?

Reply:
Oh I realize that there's source.
In that case we should change the enum. I have edited it.

Reply:
Yes, for table with connector, we may throttle source in this case.